### DISCUSSION TITLE
What is the correct field length for a68?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#57] [General]
In NOS 2.8.7 / 871's LIBD01, the default CM field length is set to all CM (10000B), if I read this right...  Everything else is below 6700B......  

Yeah, yeah, yeahhh...  I get the jokes, I have the final and dissenting reports, but it is so influential that I just have to play with it.

**Comment by kej715** [comment]
The A68 compiler appears to require a minimum field length of 100000. The
example ALGOL 68 program provided on the Cyber 865
<http://www.nostalgiccomputing.org:8080/aterm.html?m=max&t=Cyber%20865&r=45&c=80>
and Cyber 175 at the Nostalgic Computing Center
<http://www.nostalgiccomputing.org/> are compiled and executed using the
following command sequence:

RFL,100000.

A68,I=lfn,L=0.

LGO.

cheers,
Kevin

**Comment by KC5VDJ** [comment]
Ah!  Rfl did the trick, thanks!  Been trying to remember that one.
